app/models/heading.rb

```diff
@@ -106,12 +106,20 @@ def self.changes_for(depth = 0, conditions = {})
       :oid,
       :operation_date,
       :operation,
-      Sequel.as(depth, :depth)
+      Sequel.as(depth, :depth),
     ).where(conditions)
      .limit(TradeTariffBackend.change_count)
      .order(Sequel.desc(:operation_date, nulls: :last))
   end
 
+  def footnote_ids
```

Comment:
I don't understand the addition of these chipolatas

Reply:
it is used by the JSON serializer,
but I'll remove it since we don't need it.